.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+**What type of PR is this?**
+<!-- Bug, Chore, Documentation, Feature -->
+
+**What this PR does/ why we need it**:
+<!-- Explain, without going into the details, what this PR does, and what problem it solves. -->
+
+**Which issue(s) this PR fixes**:
+<!-- Add a link to the JIRA issue. Otherwise, put "no issue." -->
+
+**Special notes for your reviewer**:
+
+**Does this PR introduce a user-facing change?**:
+<!--
+If no, just write "NONE" in the release-note block below.
+If yes, a release note is required:
+Enter your extended release note in the block below. If the PR requires additional action from users switching to the new release, include the string "action required".
+-->
+```release-note
+
+```
+
+**Checklist**
+
+* [ ] The commit message explains the changes and why are needed.
+* [ ] The code builds and passes lint/style checks locally.
+* [ ] The relevant subset of e2e tests pass locally.
+* [ ] The core changes are covered by unit tests.
+* [ ] The core changes are covered by e2e tests.
```

Comment:
We only have one type of test currently, and nothing that we consider "e2e" tests at the moment:

```suggestion
```